client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,11 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
-        <div className="home-button">Home</div>
+         <Link exact to="/">
+         <div className="home-button">Home</div>
```

Comment:
```suggestion
             <div className="home-button">Home</div>
```
again needs proper indentation